index.css

```diff
@@ -1,68 +1,58 @@
 @import "https://fonts.googleapis.com/css?family=Lato:400,700";
 
+:root {
+  --body-color: white;
+  --main-color: #f5f5f0;
+  --head-color: #c2d6c1;
+  --font-color: #29a329;
+}
+
 * {
   box-sizing: border-box;
 }
 
 body {
-  background-color: white;
+  background: var(--body-color);
   margin: 0;
   font-family: "Lato", sans-serif;
 }
 
 .container {
   display: flex;
-  margin: 4rem;
-}
-
-.products-section {
-  background: #f5f5f0;
-  width: 30%;
-}
-
-.products-section__title {
-  width: 100%;
-  display: flex;
-  justify-content: center;
-  background: #b0e5ad;
-  color: #29a329;
-  font-size: 1.3rem;
-  font-weight: bold;
-  padding: .5rem;
+  padding: 4rem;
+  height: 100vh;
+  width: 100vw;
 }
 
 .main-section {
   display: flex;
   flex-direction: column;
   width: 100%;
+  margin-left: 6rem;
 }
 
-.add-product-section {
-  background: #f5f5f0;
-  margin-left: 5rem;
-  margin-bottom: 5rem;
+.section-box {
+  background: var(--main-color);
+  height: 100%;
 }
 
-.add-product-section__title {
-  display: flex;
-  justify-content: center;
-  background: #b0e5ad;
-  color: #29a329;
-  font-size: 1.3rem;
-  font-weight: bold;
-  padding: .5rem;
+.section-box--add-product {
+  margin-bottom: 5rem;
+  height: 50%
 }
 
-.summary {
-  background: #f5f5f0;
-  margin-left: 5rem;
+.section-box--products {
+  background: var(--main-color);
+  width: 30%;
 }
 
-.summary__title {
+
+
+.section-box__title {
   display: flex;
   justify-content: center;
-  background: #b0e5ad;
-  color: #29a329;
+  background: var(--head-color);
+  color: var(--font-color);
   font-size: 1.3rem;
```

Comment:
Trzymałbym się od tego projektu font-size'ów w px. Jeżeli font-size nie był modyfikowany w selektorze `html`, to 1rem == 16px. Tutaj zostałbym przy 16px, ale dałbym większy padding, powiedzmy 0.75rem;